controls/eks-cis-5.4.3.rb

```diff
@@ -21,5 +21,14 @@
   tag cis_level: 1
   tag cis_controls: ['12', 'Rev_7']
   tag cis_rid: '5.4.3'
+
+  address_key = os.windows? ? '\"ExternalIP\"' : 'ExternalIP' 
+
+  node_external_addresses = command("kubectl get nodes -o jsonpath='{.items[*].status.addresses[?(@.type==#{address_key})].address}'").stdout.strip
+
```

Comment:
I believe this check refers to the presence of `public ip` in the EKS managed EC2 nodes not he K8S configs.

So the solution could be using aws cli to query list of EC2 managed nodes within the specified EKS cluster and using another aws cli command to check if they have public IP defined.